setup.py

```diff
@@ -8,4 +8,7 @@
     test_suite='nose.collector',
     platforms=['any'],
     zip_safe=False,
+    extras_require={
```

Comment:
Don't add stuff here. This project uses `pbr`. So this should be inside `setup.cfg` as per the example on https://docs.openstack.org/developer/pbr/#environment-markers .